app/src/main/java/com/sdp/movemeet/utility/PermissionChecker.java

```diff
@@ -0,0 +1,57 @@
+package com.sdp.movemeet.utility;
```

Comment:
As we discussed, that's a very good point to have created a file gathering all the different permission checkers we need in our app, it makes our code more organized (we can now simply import the permissions in the activities requiring them)! Good job!

Reply:
+1